starky/src/transcript.rs

```diff
@@ -35,6 +35,7 @@ impl TranscriptGL {
 }
 
 impl Transcript for TranscriptGL {
+    // TODO:Check the type F is equal to F3G
```

Comment:
Would you add more details info in comment?

Reply:
If we support F5G, here we also need to add a generic F to this structure, instead of get_field. But there is a lot of efforts  to support F5G AFAK